suisa_sendemeldung/suisa_sendemeldung.py

```diff
@@ -78,7 +82,14 @@ def get_arguments(parser: ArgumentParser):  # pragma: no cover
         required=True,
     )
     parser.add_argument(
-        "--csv", env_var="CSV", help="create a csv file", action="store_true"
+        "--file", env_var="FILE", help="create file", action="store_true"
+    )
+    parser.add_argument(
+        "--filetype",
```

Comment:
we want to also add this to the readme

Reply:
done ([687dee6](https://github.com/radiorabe/suisa_sendemeldung/pull/204/commits/687dee6211cae9faa16715f435d7c2c9a1687412))